### PR TITLE
Feature/Media Skip Indicators Support

### PR DESCRIPTION
**Pull request by ShobhitAd**
Implements #113

### CODE REVIEW

File: src/js/ControlBarItem.js [hunk 1]
```diff
@@ -22,6 +24,10 @@ export default class ControlBar extends React.Component {
 
     getIcon(icon) {
         switch(icon) {
+            case "iconSkipLeft":
+                return <IconSkipLeft/>
```
[the remark this answers is not in the document]
Comment (ShobhitAd) [reply src/js/ControlBarItem.js:30]: Implemented in https://github.com/smartdevicelink/generic_hmi/pull/337/commits/15ce258f4c90f8e49959e5386b3aefc7f9f3bc04

File: src/js/ControlBarItem.js [hunk 2]
```diff
@@ -4,10 +4,14 @@ import Image from './Templates/Shared/Image'
 
 import {ReactComponent as IconSeekLeft} from '../img/icons/icon-seek-left.svg';
 import {ReactComponent as IconSeekRight} from '../img/icons/icon-seek-right.svg';
```
Comment (iCollin) [inline src/js/ControlBarItem.js:6]:
```suggestion
import {ReactComponent as IconSeekLeft} from '../img/icons/icon-seek-left.svg';
import {ReactComponent as IconSeekRight} from '../img/icons/icon-seek-right.svg';
```

these icons are no longer needed in the project, correct?

File: src/js/reducers.js [hunk 3]
```diff
@@ -473,6 +475,12 @@ function ui(state = {}, action) {
             if (action.updateMode !== "PAUSE") {
                 app.pauseTime = null
             }
+            if (action.forwardSeekIndicator) {
+                app.forwardSeekIndicator = action.forwardSeekIndicator
+            }
+            if (app.backSeekIndicator) {
```
Comment (iCollin) [inline src/js/reducers.js:481]:
```suggestion
            if (action.backSeekIndicator) {
```

**Review by iCollin — CHANGES_REQUESTED** [review]

File: src/js/reducers.js (hunk 3 above)
Comment (ShobhitAd) [reply src/js/reducers.js:481]: Good catch! Applying the change

File: src/js/ControlBarItem.js (hunk 2 above)
Comment (ShobhitAd) [reply src/js/ControlBarItem.js]: Correct. Removed in https://github.com/smartdevicelink/generic_hmi/pull/337/commits/cb0a87e8db9806fc9543502e572d3037e3669664

**Review by iCollin — APPROVED** [review]

**Review by iCollin — APPROVED** [review]

**Pull request merged**